templates/layout/default.php

```diff
@@ -42,7 +42,7 @@
         </div>
         <div class="top-nav-links">
             <a target="_blank" rel="noopener" href="https://book.cakephp.org/4/">Documentation</a>
-            <a target="_blank" rel="noopener" href="https://api.cakephp.org/4/">API</a>
+            <a target="_blank" rel="noopener" href="https://api.cakephp.org/4.1/">API</a>
```

Comment:
```suggestion
            <a target="_blank" rel="noopener" href="https://api.cakephp.org/">API</a>
```

Homepage will direct to latest minor.

Reply:
Ah yes that is a better solution then :)